bin/exec.go

```diff
@@ -4,12 +4,12 @@ import (
 	"flag"
 	"time"
 
+	"github.com/litmuschaos/chaos-executor/pkg/utils"
+	"github.com/litmuschaos/chaos-executor/pkg/utils/analytics"
+	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
```

Comment:
Again you did the same mistake, It should be in the last where it was earlier

Reply:
fixed it @chandankumar4 !